sklearn/utils/estimator_checks.py

```diff
@@ -250,7 +251,12 @@ def check_estimator(Estimator):
     name = Estimator.__name__
     check_parameters_default_constructible(name, Estimator)
     for check in _yield_all_checks(name, Estimator):
-        check(name, Estimator)
+        try:
+            check(name, Estimator)
+        except SkipTest as message:
+            # the only SkipTest thrown currently results from not
+            # being able to import pandas.
+            warnings.warn(message, ImportWarning)
```

Comment:
I'm not certain whether ImportWarning is correct. Should we have a SkipTestWarning or some such?